frontend/src/router.ts

```diff
@@ -233,26 +237,22 @@ export const checkAuth: NavigationGuard = async (to, from, next) => {
   const currentUser = firebase.auth().currentUser;
   const {
     meta: {
-      auth: { requireAuth, guestOnly, requireClaims }
+      auth: { requireAuth, guestOnly, ability }
     }
   } = restrictedRoute;
 
-  if ((requireAuth || requireClaims) && !currentUser)
+  if ((requireAuth || ability) && !currentUser)
```

Comment:
Consider using store to obtain `isSignedIn` property instead of getting current user from firebase directly.